includes/taxo.php

```diff
@@ -430,7 +430,7 @@ function ap_question_have_tags( $question_id = false ) {
  * @return bool
  */
 function is_question_tag() {
-	if ( ap_get_tag_slug() === get_query_var( 'ap_page' ) ) {
+	if ( 'tag' === ap_current_page() ) {
```

Comment:
@mebishalnapit Any reason for removing `ap_get_tag_slug()` ?

Reply:
@rahularyan We have the option `Tag page slug` which when changed did not work for displaying the breadcrumb content and setting that to the `tag` works without modifying any code. Hence, I had hard-coded it so that user might not have to code for the fix on this one. Should I revert it back to the original one since that has filter applied to it?